low-level/sensors/impl/generic/cpu_fault_sensor.py

```diff
@@ -180,8 +179,7 @@ def run(self):
                 to_update = True
 
         except Exception as e:
-            logger.error(f"Error while processing cpu info, shutting down CPUFaultSensor : {e}")
-            self.shutdown()
+            raise Exception(f"Error while processing cpu info, {e}")
```

Comment:
this won't help us to find where the actual exception happened. pls use
`raise Exception(f"Error while reading current cpu info") from e`

Reply:
As justified above.